README.md

```diff
@@ -0,0 +1,89 @@
+# recodeflow <img src="man/figures/logo.svg" align="right" width="180"/>
+
+<!-- badges: start -->
+
+[![Lifecycle:
+development](https://img.shields.io/badge/lifecycle-maturing-blue.svg)](https://www.tidyverse.org/lifecycle/#maturing)
+[![](https://img.shields.io/cran/v/cchsflow?color=green)](https://CRAN.R-project.org/package=TBA)
+![](https://img.shields.io/github/v/release/big-life-lab/recodeflow?color=green&label=GitHub)
+[![License:
+MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+<!-- badges: end -->
+
+# Introduction
+
+## What is `recodeflow`?
+
+`recodeflow` transforms variables from multiple data sets into harmonize variables.
+
+`recodeflow` has the basic functions and templates required to define, transform, and harmonize variables for any dataset.
+
+
+## Why should I use `recodeflow`? 
+
+Let's walk through an example. 
+
+Each year, Canadians complete a survey, and the data is collected. The data is coded.  You, a scientist, want to use data from the 10 surveys to answer two research questions **but there's a catch:**
```

Comment:
@Rhan43 The CCHS is an unusual example for most users and/or there are so many different use cases. I suggest keeping the example more generic. 

I'll make suggestions in the readme and commit those suggestions. 

I wonder about a specific example of how to use rec_w_table from the PBC database.

Reply:
@DougManuel I've kept the "Why use recodeflow" section very broad and generic. I reworked what you put. 

I'm not loving the idea of using an example from PBC. I had to modify the dataset for the documentation. Therefore, a generic example would still require a bit of explaining. I can attempt an example you still want one though.